perfkitbenchmarker/pkb.py

```diff
@@ -114,6 +113,12 @@
                     'characters in length.')
 flags.DEFINE_string('owner', getpass.getuser(), 'Owner name. '
                     'Used to tag created resources and performance records.')
+flags.DEFINE_boolean('enable_detailed_timing', False, 'Includes detailed '
```

Comment:
Maybe consider an enum here for different verbosities:
* `none`: No timing samples emitted.
* `end_to_end`: Only the end-to-end runtime (current behavior, default).
* `phase_runtime`: Runtime by phase
* `phase_timestamps`: Runtime by phase + timestamps.